app/views/layouts/_topbar.html.haml

```diff
@@ -1,8 +1,10 @@
 %nav.top-bar{"data-topbar" => "", :role => "navigation"}
   %ul.title-area
     %li.name
-      %h1
-        = link_to 'Shostakodech', root_path
+
+      %h1#shostakodech
+        %a#home{:href => root_path} Shostakodech
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace